public/locale/fr/config.json

```diff
@@ -60,6 +60,7 @@
     "SR5.Aoe": "Z",
     "SR5.ApplyWounds": "Appliquer les blessures",
     "SR5.Armor": "Blindage",
+    "SR5.ArmorHardened" : "Hardened",
```

Comment:
Unless the source books in these languages specifically use the word "Hardened", I'd omit this translation from everywhere but English.

Reply:
I just do it for convience for translators. If I wouldnt add it it would fall back to english anyway

Reply:
That's fair! 👍 I often see tooling for translations that looks for missing keys in the other languages as a prompt that a string needs to be translated.

Reply:
The system is registered with [https://weblate.foundryvtt-hub.com/](https://weblate.foundryvtt-hub.com/) for translations. I'd have to follow up how that community tool works with missing keys.